Trades/Trades.go

```diff
@@ -14,39 +14,71 @@ type TradeItems struct {
 }
 
 // Trade function that
-func Trade(TradeObject, TradeObjectBuffer []string) []byte {
+func Trade(tradeObjects, tradeObjectsBuffer []string) (tradeObjectsProcessed, tradeObjectsBufferProcessed []string,
+	TradedItemJSON []byte) {
+
+	//If the TradeObject slice is empty (because all previously moved to buffer) - empty buffer back to main slice
+	tradeObjects, tradeObjectsBuffer = emptyBufferTradesToMain(tradeObjects, tradeObjectsBuffer)
+
+	//Create a 'random' trade
+	TradedItem, randomObjNumber := randomTrade(tradeObjects)
```

Comment:
```suggestion
	tradedItem, randomObjNumber := randomTrade(tradeObjects)
```

typically we write private variables defined within a function in camel case.